config/environments/production.rb

```diff
@@ -68,13 +76,13 @@
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.perform_deliveries = true
   config.action_mailer.smtp_settings = {
-    address: 'smtp.mandrillapp.com',
-    port: 587,
+    address: ENV['MAIL_SMTP_HOST'] || MAIL_DEFAULTS[:host],
+    port: ENV['MAIL_SMTP_PORT'] || MAIL_DEFAULTS[:smtp_port],
     user_name: ENV['MAIL_USER_NAME'],
     password: ENV['MAIL_PASSWORD'],
-    authentication: :plain,
-    enable_starttls_auto: true,
-    domain: ENV['DOMAIN']
+    authentication: (ENV['MAIL_SMTP_AUTH_METHOD'] || MAIL_DEFAULTS[:smtp_auth_method]).to_sym,
```

Comment:
let's name MAIL_DEFAULST key exactly the same as it is accepted here -- `: authentication `